react/features/welcome/components/AbstractWelcomePage.js

```diff
@@ -4,6 +4,7 @@ import { appNavigate } from '../../app';
 import { isRoomValid } from '../../base/conference';
 import { VideoTrack } from '../../base/media';
 import { getLocalVideoTrack } from '../../base/tracks';
+import { RoomnameGenerator as RoomnameGen } from '../../base/util';
```

Comment:
Using multiple names for one and the same entity is against our coding style. Please don't use a second name for the same entity and comply with our coding style.